src/iuse.cpp

```diff
@@ -954,11 +954,12 @@ int iuse::flusleep(player *p, item *it, bool, const tripoint& )
 
 int iuse::inhaler(player *p, item *it, bool, const tripoint& )
 {
-    p->remove_effect( effect_asthma);
-    p->add_msg_if_player(m_neutral, _("You take a puff from your inhaler."));
-    if (one_in(50)) {  // adverse reaction
-        p->add_msg_if_player(m_bad, _("Your heart begins to race."));
-        p->mod_fatigue(-10);
+    p->add_msg_if_player( m_neutral, _( "You take a puff from your inhaler." ) );
+    if( !p->remove_effect( effect_asthma) ) {
+        p->mod_fatigue( -10 ); // if we don't have asthma can be used as stimulant
```

Comment:
The `one_(50)` change was intentional (spamming 50 keypresses to get an effect isn't useful) however it looks like it became a little too good.

I think `shakes` is quite an appropriate effect. How about reducing the `fatigue` bonus from 10 to 2 and keeping the same risk (1:20) of an adverse effect (`shakes`)?